modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -24,68 +25,51 @@ about the update.
-the v5 breaking code changes. Simply run:
+We've introduced a [new codemod package](./modules/codemod) you can use to automatically update your
+code to work with a majority of the breaking changes in the migration from Canvas Kit v4 to v5.
+Simply run:
 
 ```sh
 > npx @workday/canvas-kit-codemod v5 [path]
 ```
 
-> Note: This codemod only work on .js, .jsx, .ts, and .tsx extensions. You may need to make some
-> manual changes in other file types (.json, .mdx, .md, etc.).
+> Note: This codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` extensions. You may need to make
+> some manual changes in other file types (`.json`, `.mdx`, `.md`, etc.).
 
 > Note: You may need to run your linter after executing the codemod, as it's resulting formatting
 > (spacing, quotes, etc.) may not match your project's styling.
 
-Breaking changes accounted for by this codemod will be marked with a 🤖.
+**Breaking changes accounted for by this codemod will be marked with a 🤖.**
 
-Please [let us know](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if
-you have any troubles or missed use cases with this codemod. The `@workday/canvas-kit-codemod`
-package will help us maintain additional codemod transforms to make future migrations easier.
+Please verify all changes made by the codemod, and
+[let us know](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you
+encounter any issues or use cases that we've missed. The `@workday/canvas-kit-codemod` package will
+help us maintain additional codemod transforms to make future migrations easier.
 
-### Rename Core and Labs Core Modules to Tokens
+## General Changes
 
-The distinction between our core and common packages is often unclear and creates confusion around
-what should be imported from where. To help alleviate this and better align with our design
-taxonomy, we're renaming our `core` packages to `tokens`. We are also removing the `type` default
-\export the labs core package in favor of named imports. These changes are listed below; all of
-which are handled by the v5 codemod.
+### Compound Components
 
-#### Automatic Updates
+Components are transitioning to [Compound Components](./COMPOUND_COMPONENTS.mdx):
 
-- 🤖 Core import statements
-- e.g. `import { space } from '@workday/canvas-kit-react/core';` becomes
-  `import { space } from '@workday/canvas-kit-react/tokens';`
-- 🤖 Labs core import statements
-- e.g. `import { type } from '@workday/canvas-kit-labs-react/core';` becomes
-  `import { type } from '@workday/canvas-kit-labs-react/tokens';`
-- 🤖 Labs core default type import
-- e.g. `import type from '@workday/canvas-kit-labs-react/core';` becomes
-  `import { type } from '@workday/canvas-kit-labs-react/tokens';`
+- Components will support [forwarded refs](https://reactjs.org/docs/forwarding-refs.html)
+- The corresponding tag of a component can be changed with the `as` prop
```

Comment:
The `Compound Components` section could stand to be filled out a bit more for emphasis. Maybe not to the extent of the release notes/press release, but we could lift some material from there.